src/Web/Rsp.IrasPortal.Web/Controllers/ReviewBodyController.cs

```diff
@@ -302,4 +306,151 @@ public async Task<IActionResult> AuditTrail(Guid reviewBodyId, int pageNumber =
 
         return View(AuditTrailView, resultModel);
     }
+
+    /// <summary>
+    ///     Displays users for a review body
+    /// </summary>
+    [HttpGet]
+    [Route("/reviewbody/users/{id}", Name = "rbc:viewreviewbodyusers")]
```

Comment:
Route names are generated by the following line on the controller
`[Route("[controller]/[action]", Name = "rbc:[action]")]`

So you can just specify the parameters needed for the route, and it will be merged automatically with the Route Template.

```suggestion
    [Route("{id}"]
```